sqlmesh/core/snapshot/definition.py

```diff
@@ -240,6 +240,10 @@ def data_version(self) -> SnapshotDataVersion:
     def is_new_version(self) -> bool:
         raise NotImplementedError
 
+    @property
+    def model_fqt(self) -> t.Optional[exp.Table]:
```

Comment:
Ditto: confusing term. I don't think fqt is a thing

Reply:
![fqt](https://github.com/TobikoData/sqlmesh/assets/6326532/09ee15f1-b4e0-42dc-a846-f1b4ac309074)

Reply:
![fqt](https://github.com/TobikoData/sqlmesh/assets/6326532/836bfe83-c0f1-407d-9796-d5976d86a891)